modules/test/playwright/tests/commerce/commercePayment.spec.js

```diff
@@ -11,7 +11,12 @@ import {test as commercePagesTest} from '../../fixtures/commercePages.fixture';
 import {test as dataHelperTest} from '../../fixtures/dataHelper.fixture';
 import {getRandomInt} from '../../utils/util';
 
-export const test = mergeTests(ApiHelpersTest, applicationsMenuPagesTest, commercePagesTest, dataHelperTest);
+export const test = mergeTests(
```

Comment:
There is no need to commit this test, please remove it and everything that refers to this test that is not needed for your test cases